k8s/namespaces/fees-pay/refund-notifications-job/demo.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: refund-notifications-job
+  namespace: fees-pay
+spec:
+  releaseName: refund-notifications-job
```

Comment:
Can you remove duplication` releaseName: refund-notifications-job`

Reply:
done